src/src/Helpers/StringConverter.cpp

```diff
@@ -1317,9 +1317,21 @@ void parseSingleControllerVariable(String            & s,
 
 void parseSystemVariables(String& s, bool useURLencode)
 {
+  String MaskEscapedPercent = "";

```

Comment:
Strings are created with empty content by default, no need to initialize them with the (undesired) plain C string "" value.

Reply:
Done!